pkg/user/user.go

```diff
@@ -131,10 +131,17 @@ func ReconcileRemove(u *iam.User, organization, orderer string, idType IDType) (
 	if err != nil {
 		return BlockchainAnnotation{}, errors.Wrap(err, "organization "+organization)
 	}
-	// delete organization from user's annotation
-	err = annotationList.DeleteAnnotation(organization)
-	if err != nil && err != ErrAnnotationNotExist {
-		return BlockchainAnnotation{}, err
+	if orderer != "" {
```

Comment:
should consider  IDType `peer` (same as orderer) in `ReconcileAdd` `ReconcileRemove`

Reply:
Done.